stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt

```diff
@@ -211,4 +229,7 @@ class ChatFragment : Fragment() {
         }
         return previousYear != year || previousDayOfYear != dayOfYear
     }
+
+    private fun Result<Flag>.isAlreadyExistsError(): Boolean =
+        isError && error().message?.contains("flag already exists") == true
```

Comment:
There is no easy way to hide this option from menu so we allow to flag the message multiple times

Reply:
Maybe ask BE team? Because they can change message text easily in the future and our implementation will get broken

Reply:
I think we can rely on status codes instead. I'll double-check values with backend team tomorrow